src/include/daos_srv/bio.h

```diff
@@ -30,93 +30,8 @@
 #define __BIO_API_H__
 
 #include <daos/mem.h>
-#include <daos/common.h>
 #include <abt.h>
-
-typedef struct {
-	/*
-	 * Byte offset within PMDK pmemobj pool for SCM;
-	 * Byte offset within SPDK blob for NVMe.
-	 */
-	uint64_t	ba_off;
-	/* DAOS_MEDIA_SCM or DAOS_MEDIA_NVME */
-	uint16_t	ba_type;
-	/* Is the address a hole ? */
-	uint16_t	ba_hole;
-	uint16_t	ba_dedup;
-	uint16_t	ba_padding;
-} bio_addr_t;
-
-/** Ensure this remains compatible */
-D_CASSERT(sizeof(((bio_addr_t *)0)->ba_off) == sizeof(umem_off_t));
-
-struct bio_iov {
-	/*
-	 * For SCM, it's direct memory address of 'ba_off';
-	 * For NVMe, it's a DMA buffer allocated by SPDK malloc API.
-	 */
-	void		*bi_buf;
-	/* Data length in bytes */
-	size_t		 bi_data_len;
-	bio_addr_t	 bi_addr;
-
-	/** can be used to fetch more than actual address. Useful if more
-	 * data is needed for processing (like checksums) than requested.
-	 * Prefix and suffix are needed because 'extra' needed data might
-	 * be before or after actual requested data.
-	 */
-	size_t		 bi_prefix_len; /** bytes before */
-	size_t		 bi_suffix_len; /** bytes after */
-};
-
-struct bio_sglist {
-	struct bio_iov	*bs_iovs;
-	unsigned int	 bs_nr;
-	unsigned int	 bs_nr_out;
-};
-
-/* Opaque I/O descriptor */
-struct bio_desc;
-/* Opaque I/O context */
-struct bio_io_context;
-/* Opaque per-xstream context */
-struct bio_xs_context;
-struct bio_blobstore;
```

Comment:
I don't see why "bio_blobstore" was exported, please remove it from header if the patch needs be refreshed.